concepts/05 UI Components/Lookup/00 Getting Started with Lookup/15 Configure Search.md

```diff
@@ -0,0 +1,68 @@
+The Lookup allows users to search through the drop-down list items. To configure the search, set the following properties:
+
+- [searchMode]()    
+This property uses one of the two search modes: "contains" or "startswith".  
+
+- [searchExpr]()    
+You can specify the data source item field or several field values in an array to define the search criteria.
+
+- [minSearchLength]()    
+Specify the minimal search length with this property.
+
+To disable the search, set the [searchEnabled]() property to **false**.
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#{widgetName}").dx{WidgetName}({
```

Comment:
widgetName and other placeholders are signs that an article is probably reused somewhere else, and a techwriter that will edit it in the future should keep that in mind. It is better to refrain from using placeholders if an article is not reused, as in this case.

Reply:
Noted